test/tools/dshell_prebuilt/dshell_prebuilt.d

```diff
@@ -289,3 +289,14 @@ private GrepResult grepLines(T)(T lineRange, string finalPattern)
     writefln("[GREP] matched %s lines", matches.data.length);
     return GrepResult(matches.data);
 }
+
+/**
+read the the given `file` and remove \r and the compiler debug header.
+*/
+string readOutput(string file)
```

Comment:
Instead of a function to read a file of compiler output, how about a function that called 'filterCompilerDebugMsg'?

Reply:
Good idea, I'll change it.